docs/documentation.md

```diff
@@ -79,13 +80,16 @@ Platform versions follow standard numbering, and releases below 1 should be cons
 #### List of versions (newest on top):
 
 - [release 1]()
+- [legacy branch](https://github.com/NorESMhub/noresm-land-sites-platform/tree/legacy) - for simulations with input data generated with [NorESMhub/noresm-lsp-input](https://github.com/NorESMhub/noresm-lsp-input) repo. Use with `legacy` tags of [ctsm-api](https://github.com/NorESMhub/ctsm-api/releases/tag/legacy) and [noresm-lsp-ui](https://github.com/NorESMhub/noresm-lsp-ui/releases/tag/legacy)
 - [archive tag](https://github.com/NorESMhub/noresm-land-sites-platform/releases/tag/archive)
 - [tag 0.1.0-dev](https://github.com/NorESMhub/noresm-land-sites-platform/releases/tag/v0.1.0)
 
 #### Model versions
 
-The platform is primarily designed to run the land model (CLM) with the Norwegian Earth System Model. The versions of FATES and CLM therefore have to be in line with stable NorESM versions. NorESM is integrated into the platform via a dedicated `noresm_landsites` branch in the [NorESMhub/NorESM repository](https://github.com/NorESMhub/NorESM/tree/noresm_landsites).
+The platform is primarily designed to run the land model (CLM) with the Norwegian Earth System Model. The versions of FATES and CLM therefore have to be in line with stable NorESM versions. You can see the component model versions in the `Externals.cfg` file under `/resources/overwrites/` in the [noresm-land-sites-platform repository](https://github.com/NorESMhub/noresm-land-sites-platform/tree/main/resources/overwrites).
 
+##### Changing the model version
+To change the model versions, either to update the NorESM-LSP or on your own fork, there are two files to consider. The first is the NorESM version in the Dockerfile (`noresm-land-sites-platform/docker/api/.env`), and the second (more important) is a file to overwrite the model component versions located in `NorESM-land-sites-platform/resources/overwrites/Externals.cfg`. The FATES version is defined in `NorESM-land-sites-platform/resources/model/components/clm/Externals_CLM.cfg`.
```

Comment:
Instead of "The first is the NorESM version in the Dockerfile", we should say something like "The first is the NorESM version in the docker environment file".

We should also mention that changing FATES version requires a manual run of "checkout_externals".
Alternatively, they can point to a version of CTSM in `Externals.cfg` that uses the desired version of FATES.

Reply:
fixed